mlrun/execution.py

```diff
@@ -944,6 +959,29 @@ def set_if_valid(struct, key, val):
         self._data_stores.to_dict(struct["spec"])
         return struct
 
+    def _get_updates(self):
+        def set_if_valid(struct, key, val):
+            if val:
+                struct[key] = val
```

Comment:
just to separate the variables for readability
```suggestion
        def set_if_valid(_struct, key, val):
            if val:
                _struct[key] = val
```

Reply:
maybe also call the method `set_if_not_none`